Set onkyo max volume to 200 for newer receivers

libots opened the pull request:
## Breaking Change:

Not so much a breaking issue, but older receivers that had a max volume of 80 will now never read to HA as at max volume. This can be fixed by adding in ```max_volume = 80``` into their configuration.

## Description:

Newer Onkyo receivers (e.g, TX-RZ830) volume ranges from 0-100 in increments of 0.5, and reports the volume to HA as 0-200.

**Related issue (if applicable):** fixes #<home-assistant issue number goes here>

**Pull request with documentation for [home-assistant.io](https://github.com/home-assistant/home-assistant.io) (if applicable):** home-assistant/home-assistant.io#<home-assistant.io PR number goes here>

## Example entry for `configuration.yaml` (if applicable):
```yaml

```

## Checklist:
  - [X] The code change is tested and works locally.
  - [x] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [x] There is no commented out code in this PR.
  - [x] I have followed the [development checklist][dev-checklist]

If user exposed functionality or configuration variables are added/changed:
  - [x]\Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)

If the code communicates with devices, web services, or third-party tools:
  - [x] [_The manifest file_][manifest-docs] has all fields filled out correctly. Update and include derived files by running `python3 -m script.hassfest`.
  - [x] New or updated dependencies have been added to `requirements_all.txt` by running `python3 -m script.gen_requirements_all`.
  - [x] Untested files have been added to `.coveragerc`.

If the code does not interact with devices:
  - [ ] Tests have been added to verify that the new code works.

[dev-checklist]: https://developers.home-assistant.io/docs/en/development_checklist.html
[manifest-docs]: https://developers.home-assistant.io/docs/en/creating_integration_manifest.html

libots committed:
Max volume to 200 for newer receivers

Newer Onkyo receivers (e.g, TX-RZ830) volume ranges from 0-100 in increments of 0.5, and reports the volume to HA as 0-200. If this is controversial, default CONF_MAX_VOLUME can be changed to the old value of 80.

homeassistant commented:
Hi @libots, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant#26341).

Once you do that we will be able to review and accept this pull request.

Thanks!

MartinHjelmare reviewed (changes requested):
Please update the docstring for `set_volume_level` entity method too. It mentions 80 as max level.

libots committed:
Update all volume functions

Update all volume functions (and docstrings) from 80 to 200 to ensure consistency when max_volume is not set manually.

libots committed:
Fix pydocstyle error

libots committed:
Fix pydocstyle error

blank line whitespace
